Curve-finance-suite/Unkill/README.md

```diff
@@ -0,0 +1,20 @@
+# UnkillMe Agent
```

Comment:
Could include a tx hash to test the agent with mainnet data.

Reply:
Sorry, I wasn't able to find on relevant transaction.